### PR TITLE
Number of bits to be flipped to inter convert two numbers Documentation

### LINKED ISSUE #274: Number of bits to be flipped to inter convert two numbers Documentation
## 💥 Proposal

Number of bits to be flipped to inter convert two numbers

### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#reporting-new-issues)?

YES

### PR DESCRIPTION
**Pull request by Harikrishnan6336**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

YES

### Description

Number of bits to be flipped to inter convert two numbers Documentation

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

Fixes #274

### CODE REVIEW

File: docs/bit-manipulation/no-of-bits-to-be-flipped-to-interconvert-two-numbers.mdx [hunk 1]
```diff
@@ -0,0 +1,142 @@
+---
+title: Number of bits to be flipped to interconvert two numbers
```
Comment (HarshCasper) [inline docs/bit-manipulation/no-of-bits-to-be-flipped-to-interconvert-two-numbers.mdx:2]: Use a shorter title

File: docs/bit-manipulation/no-of-bits-to-be-flipped-to-interconvert-two-numbers.mdx [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+slug: /bit-manipulation/no-of-bits-to-be-flipped-to-interconvert-two-numbers
```
Comment (HarshCasper) [inline docs/bit-manipulation/no-of-bits-to-be-flipped-to-interconvert-two-numbers.mdx:3]: Use a more compact slug

File: docs/bit-manipulation/no-of-bits-to-be-flipped-to-interconvert-two-numbers.mdx [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+---
+
+## Introduction
+
+Given two numbers we have to find the number of bits to be flipped to convert one number to another.
```
Comment (HarshCasper) [inline docs/bit-manipulation/no-of-bits-to-be-flipped-to-interconvert-two-numbers.mdx:8]: Add a bit more context

File: docs/bit-manipulation/no-of-bits-to-be-flipped-to-interconvert-two-numbers.mdx [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+```
+Binary equivalent of n1 is 00001100
+Binary equivalent of n2 is 00010001
+
+Here, we need to flip the first, third, forth and fifth bits from the LSB of n1 to convert it to n2.
+Hence, the number of bits to be flipped to interconvert the given numbers is 4.
+```
+
+We need to flip the bits, present in same places in both numbers, that are not the same.
+We can find such bits by taking an **xor** of the given numbers and counting the number of set bits in them.
```
Comment (HarshCasper) [inline docs/bit-manipulation/no-of-bits-to-be-flipped-to-interconvert-two-numbers.mdx:23]: wrap in a single line

**Review by HarshCasper — CHANGES_REQUESTED** [review]

File: docs/bit-manipulation/count-bits-flipped-to-convert-two-numbers.mdx [hunk 5]
```diff
@@ -0,0 +1,141 @@
+---
+title: Count number of bits to be flipped to convert two numbers
```
Comment (HarshCasper) [inline docs/bit-manipulation/count-bits-flipped-to-convert-two-numbers.mdx:2]: Adopt a shorter title and file name

File: docs/bit-manipulation/count-bits-flipped-to-convert-two-numbers.mdx [hunk 6]
```diff
@@ -0,0 +1,141 @@
...
+slug: /bit-manipulation/count-bits-flipped-to-convert-two-numbers
+---
+
+## Introduction
+
+Given two numbers we have to find the number of bits to be flipped to convert one number to another. We have to count the number of bits that are present in same places but is different in value in both the numbers.
```
Comment (HarshCasper) [inline docs/bit-manipulation/count-bits-flipped-to-convert-two-numbers.mdx:8]: Can you add an example here as well?

**Review by HarshCasper — CHANGES_REQUESTED** [review]

**Review by ankitaggarwal23 — COMMENTED** [review]
Closing this due to inactivity. Thank you for your valuable time.😊

**Pull request closed without merging**